README.md

```diff
@@ -48,22 +48,27 @@ there as well.
 
 In earlier HTML versions, `iframe`s were put inside of borders by default.
 Many people found that ugly. The `frameborder` allowed us to turn the border
-off.  Nowadays we like to control that kind of thing with CSS and `frameborder`
-is consider _deprecated_ meaning "likely to be removed from the standard."
+off.  
 
-We see this appear often in `iframe` links as providers try to support _as many
-browsers as possible_. It's important to understand what it means, but you
-probably won't need to write it much.  If you do, you'll want to set both
-`frameborder="0"` AND `style="border:0"`.
+The `frameborder` attribute is considered _deprecated_, meaning "likely to be
+removed from the standard." In modern browsers, we can control borders _just_
+using CSS, as with our example, `style="border:0"`.
```

Comment:
At this point they haven't seen CSS yet. Could we add something like "you might not be familiar with CSS yet. For the time being feel free to simply add this attribute"

Could you work with that idea a bit? Basically don't assume they _know_ css at this point.